api/src/Handler/BookHandler.php

```diff
@@ -73,13 +68,14 @@ public function __invoke(Book $book): void
             return;
         }
 
-        $imageUrl = $contents['data']['memes'][\mt_rand(0, 99)]['url'];
+        $imageUrl = $contents['data']['memes'][\random_int(0, 99)]['url'];
```

Comment:
`random_int` renders a cryptographically secure value, but is slower than `mt_rand`. In this case, we don't need a secured value, so I guess it's fine to use `mt_rand` here